src/frequenz/sdk/microgrid/_data_pipeline.py

```diff
@@ -103,16 +105,19 @@ def __init__(
         self._battery_power_wrapper = PowerWrapper(
             self._channel_registry,
             api_power_request_timeout=api_power_request_timeout,
+            power_manager_algorithm=Algorithm.SHIFTING_MATRYOSHKA,
             component_category=ComponentCategory.BATTERY,
         )
         self._ev_power_wrapper = PowerWrapper(
             self._channel_registry,
             api_power_request_timeout=api_power_request_timeout,
+            power_manager_algorithm=Algorithm.MATRYOSHKA,
             component_category=ComponentCategory.EV_CHARGER,
         )
         self._pv_power_wrapper = PowerWrapper(
             self._channel_registry,
             api_power_request_timeout=api_power_request_timeout,
+            power_manager_algorithm=Algorithm.MATRYOSHKA,
             component_category=ComponentCategory.INVERTER,
             component_type=InverterType.SOLAR,
         )
```

Comment:
If I remember correctly, original Matryoshka algorithm had something like `operating point` adjustment.
How it works now?

Reply:
That was like a partition, so all actors that were in the operating point group would shift the other actors or something like that. I think now is like the operating point group is always empty. But maybe @shsms can check I'm not mistaken.

Reply:
That was mainly for battery use cases and is gone now.  We only have a single matryoshka instance for PV and EV chargers.

Reply:
OK, thanks a lot! :)